tdrs-backend/tdpservice/scheduling/parser_task.py

```diff
@@ -27,6 +27,8 @@ def parse(data_file_id):
 
     if "Case Data" in data_file.section:
```

Comment:
not related to this PR but we should have had "Case Data" as a CONSTANT, and I would put it in .DataFile.py